src/least_cost_path/least_cost_path.py

```
@@ -1,21 +1,23 @@
 import math
+import os
 import time
 from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)